test/python_tests/sqlite_test.py

```diff
@@ -469,7 +469,7 @@ def test_that_64bit_int_fields_work():
     def test_null_id_field():
         # silence null key warning: https://github.com/mapnik/mapnik/issues/1889
         default_logging_severity = mapnik.logger.get_severity()
-        mapnik.logger.set_severity(mapnik.severity_type.None)
+        mapnik.logger.set_severity(getattr(mapnik.severity_type, "None"))
```

Comment:
Yes, it breaks in python 3, and the best fix is to change the name of the variable, but until now I've just changed the python code, not the C++ one. But the good move is renaming it, and keeping backward compatibility.

I wanted to see if I was able to fix all test with python only changes, before starting changes in the C++ code.